googlesource/googlesource.go

```diff
@@ -31,6 +31,11 @@ import (
 	"google.golang.org/api/option"
 )
 
+const (
+	LIMIT      = " LIMIT "
+	SelectFrom = "SELECT * FROM `"
```

Comment:
The linter said we should do this, right? 🙈 Ok, let's leave it, but for the record, this doesn't make the code more readable or maintainable 🙃